specification/purview/Azure.Analytics.Purview.DataMap/routes.tsp

```diff
@@ -0,0 +1,2084 @@
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  @doc("Update classification on an entity identified by its type and unique attributes.")
+  @route("/atlas/v2/entity/uniqueAttribute/type/{typeName}/classifications")
+  @put
+  updateClassificationsByUniqueAttribute is AtlasBaseOperation<
+    {
+      @doc("The name of the type.")
+      @path
+      typeName: string;
+
+      @doc("""
+The qualified name of the entity. (This is only an example. qualifiedName can
+be changed to other unique attributes)
+""")
+      @query("attr:qualifiedName")
+      attr?: string;
+
+      @doc("An array of classification to be updated.")
+      @body
+      atlasClassificationArray: AtlasClassification[];
+    },
+    void
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  @doc("Set classifications on entities in bulk.")
+  @route("/atlas/v2/entity/bulk/setClassifications")
+  @post
+  bulkSetClassifications is AtlasBaseOperation<
+    {
+      @doc("Atlas entity headers.")
+      @body
+      entityHeaders: AtlasEntityHeaders;
+    },
+    OkResponse<string[]>
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  #suppress "@azure-tools/typespec-azure-core/casing-style" "This is Atlas API behavior"
+  @doc("""
+Bulk API to retrieve list of entities identified by its unique attributes.
+In
+addition to the typeName path parameter, attribute key-value pair(s) can be
+provided in the following
+format
+
+typeName=\\<typeName>&attr_1:\\<attrName>=\\<attrValue>&attr_2:\\<attrName>=\\<attrValue>&attr_3:\\<attrName>=\\<attrValue>
+
+NOTE:
+The attrName should be an unique attribute for the given entity-type.
+The REST
+request would look something like this
+
+GET
+/v2/entity/bulk/uniqueAttribute/type/hive_db?attr_1:qualifiedName=db1@cl1&attr_2:qualifiedName=db2@cl1
+
+Note:
+at least one unique attribute must be provided.
+""")
+  @route("/atlas/v2/entity/bulk/uniqueAttribute/type/{typeName}")
+  @get
+  listByUniqueAttributes is AtlasBaseOperation<
+    {
+      @doc("The name of the type.")
+      @path
+      typeName: string;
+
+      @doc("Whether to return minimal information for referred entities.")
+      @query
+      minExtInfo?: boolean;
+
+      @doc("Whether to ignore relationship attributes.")
+      @query
+      ignoreRelationships?: boolean;
+
+      @doc("""
+Qualified name of an entity. E.g. to find 2 entities you can set
+attrs_1:qualifiedName=db1@cl1&attrs_2:qualifiedName=db2@cl1. (This is only an
+example. qualifiedName can be changed to other unique attributes)
+""")
+      @query
+      `attr_N:qualifiedName`?: string;
+    },
+    AtlasEntitiesWithExtInfo
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "This is Atlas API"
+  @doc("Get entity header given its GUID.")
+  @route("/atlas/v2/entity/guid/{guid}/header")
+  @get
+  getHeader is AtlasBaseOperation<
```

Comment:
Can this use a ResourceRead template?

Reply:
The api path does not fit into the template.